build-system/tasks/check-links.js

```diff
@@ -146,6 +146,10 @@ function filterWhitelistedLinks(markdown) {
   filteredMarkdown =
       filteredMarkdown.replace(/(\(|\[)?http:\/\/localhost:8000/g, '');
 
+  // cdn.ampproject.org link
```

Comment:
See the error message in https://travis-ci.org/ampproject/amphtml/jobs/261912769